velox/functions/lib/Re2Functions.cpp

```diff
@@ -817,6 +1055,89 @@ std::shared_ptr<VectorFunction> makeRe2Extract(
   }
 }
 
+std::vector<std::shared_ptr<exec::FunctionSignature>> re2ReplaceSignatures(){
+  return {
+
+    // regex_replace (str, pattern, replace)
+    exec::FunctionSignatureBuilder()
+          .returnType("varchar")
+          .argumentType("varchar")
+          .argumentType("varchar")
+          .argumentType("varchar")
+          .build(),
+    // regex_replace (str, pattern, replace, position)
+    exec::FunctionSignatureBuilder()
+          .returnType("varchar")
+          .argumentType("varchar")
+          .argumentType("varchar")
+          .argumentType("varchar")
+          .argumentType("integer")
+          .build(),
+        // regex_replace (str, pattern, replace, position)
```

Comment:
Please refer [doc](https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md) for code style and format.